bigbluebutton-html5/private/locales/en.json

```diff
@@ -60,6 +60,11 @@
     "app.userList.userOptions.unmuteAllDesc": "Unmutes the meeting",
     "app.userList.userOptions.lockViewersLabel": "Lock viewers",
     "app.userList.userOptions.lockViewersDesc": "Lock certain functionalities for attendees of the meeting",
+    "app.userList.userOptions.disableCam": "Lock rule to disable cam was enabled",
+    "app.userList.userOptions.disableMic": "Lock rule to disable microphone was enabled",
+    "app.userList.userOptions.disablePrivChat": "Lock rule to disable private chat was enabled",
+    "app.userList.userOptions.disablePubChat": "Lock rule to disable public chat was enabled",
```

Comment:
The strings could use some work. Something like, "Public chat is disabled", says the same thing in half the words.

Reply:
Sure. I'll do it.